src/partials/messenger.conversation.html

```diff
@@ -20,6 +20,9 @@
             <eee-verification-level ng-if="ctrl.type == 'contact'"
                                     contact="ctrl.receiver"></eee-verification-level>
             </div>
+            <div class="conversation-header-details-detail" ng-if="ctrl.type == 'group'">
```

Comment:
please add the whole list to the `title` attribute

    title="{{ctrl.receiver.members | idToName}}"